src/build.jl

```diff
@@ -82,22 +92,22 @@ function build(m::Model)
         price_input1_name = s.inputs[1].commodity
         price_input2_name = s.inputs[2].commodity
 
-        exp_1 = s.inputs[1].quantity/s.output_quantity
-        exp_2 = s.inputs[2].quantity/s.output_quantity
+        exp_1 = :($(swap_our_param_with_jump_param(s.inputs[1].quantity))/$(s.output_quantity))
+        exp_2 = :($(swap_our_param_with_jump_param(s.inputs[2].quantity))/$(s.output_quantity))
```

Comment:
So presumably we are going to support expressions in `output_quantity` as well down the road? In that case it would actually be easier to call `swap_our_param_with_jump_param` on the entire expression, rather than calling it separately for each interpolated value. So you could write `exp_1 = swap_our_param_BLABLA( :( BLABLA ) )`.

Reply:
That works now.